.github/workflows/security.yaml

```diff
@@ -0,0 +1,79 @@
+---
+name: security
+
+on:
+  push:
+    branches: [development/*, stabilization/*, hotfix/*]
+  release:
+    types: [published]
+  pull_request:
+    branches: [development/*, stabilization/*, hotfix/*]
+  schedule:
+    - cron: '0 8 * * 1' # Monday - 8am - UTC
+  workflow_dispatch:
+
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    permissions:
+      contents: read
+      packages: write
+    steps:
```

Comment:
as this repo is not using any kind of internally made reuasable workflow, feel free to use the one on scality/workflows.